plugins/flutter_aepedgeidentity/pubspec.yaml

```diff
@@ -0,0 +1,36 @@
+name: flutter_aepedgeidentity
+
+description: Official Adobe Experience Platform support for Flutter apps. The Experience Platform Edge Identity extension enables sending data to the Adobe Experience Edge from a mobile device.
```

Comment:
enables handling of user identity data from a mobile app when using the Adobe Experience Platform SDK and the Edge Network extension.